### ISSUE TITLE
Creating a chapter at the end of the timelime cannot be selected/deleted

### ISSUE THREAD

**Issue by Zalunda** [OpenFunscripter/OFS#69]
I was creating the last chapter and I selected the end of a video and chose "Create Chapter" by mistake, instead of "Set Size". After that, I could see the chapter to the right of the timeline but I couldn't select it to delete it.

![image](https://user-images.githubusercontent.com/8295521/209007297-7dbb0ce5-1926-4277-9f9c-64f180ecac4b.png)

Also, not related but, a little nitpicking, for the rotating colors, it would be good to only use darker colors with the currently white text, or change to lighter colors with black text. Or use white/black text automatically according to the background color.

Right now, some colors are hard to read.

![image](https://user-images.githubusercontent.com/8295521/209007487-755ddceb-5f77-4444-9ab9-280174af950d.png)

**Comment by OpenFunscripter** [comment]
Yes thanks for reporting this.

> Also, not related but, a little nitpicking, for the rotating colors, it would be good to only use darker colors with the currently white text, or change to lighter colors with black text

Nitpicking is good because I'm not nitpicky enough.
But the colors being to bright is something I noticed shortly after releaseing 3.1.0 myself.
I might've forgotten though without this so again nitpicking is good.
I'll make them darker.

**Comment by OpenFunscripter** [comment]
I added a window with a table containing all chapters to complement the current UI.
It doesn't do much but it gives you the ability to delete the chapter created outside of the video duration.

**Comment by Zalunda** [comment]
Thanks, the new table of chapters worked well. It might also be useful to "batch update" text/colors.

**Comment by OpenFunscripter** [comment]
I don't think batch updates are necessary.
You can just ctrl+c ctrl+v and updating colors is also not very hard because you can drag'n drop. 😎
And if you create hundreds of chapters that's on you I don't want to support that.

![Animation](https://user-images.githubusercontent.com/86110971/209202454-3c39d0d2-df88-4f24-b589-2eff8277e710.gif)

**Comment by Zalunda** [comment]
No, no, I didn't mean to have features to batch update. I just meant that it's a little bit easier to change manually colors / chapter names in that windows than with the "right click" on the bar in the main UI.

**Comment by OpenFunscripter** [comment]
Oh yes sorry I misread that.
